.github/ISSUE_TEMPLATE/request_gsd_id.yml

```diff
@@ -0,0 +1,8 @@
+name: GSD ID Request
```

Comment:
Shouldn't we use the web form for this?

Reply:
We should direct folks to the web form, yeah. I thought that if someone is already on GitHub and wants to make an issue, I might as well let them make a GSD straight from it. Maybe it's better to just see if we can auto direct them to the web form instead?